accounts/views.py

```diff
@@ -12,8 +15,16 @@ class IndexView(TemplateView):
     template_name = "accounts/index.html"
 
 
-class HomeView(TemplateView):
+class HomeView(LoginRequiredMixin, ListView):
     template_name = "accounts/home.html"
+    context_object_name = "tweet_list"
+
+    def get_queryset(self):
+        return Tweet.objects.all().select_related("user")
```

Comment:
N+1解消出来ていてgoodです！
これだけであれば，
```suggestion
    context_object_name = "tweet_list"
    queryset = Tweet.objects.all().select_related("user")
```
でもOKです